### ISSUE TITLE
Sass precompilation

### ISSUE THREAD

**Issue by Cherepoc** [w8tcha/BootstrapEmail.Net#45]
Hi!
This library uses file caches for styles that it compiles and creates in the runtime. I presume it was made that way for cli usage primarily, which makes total sense. Unfortunately in my setup it's not possible to precompile templates, as they are rendered with razor components, with layouts and child components. However using cli partially, to precompile sass would be a great thing. I wonder if it's possible with this library.
Thanks!

**Comment by w8tcha** [comment]
If I understand correctly you want to use css files instead if sass files, and skip the sass compiling. Correct?

Right now this is not possible, but I can add this as option.

**Comment by Cherepoc** [comment]
I still want to use sass files, I need custom styles in my emails. But instead of compiling sass and getting the css files in the runtime I want that to happen in compile time, or buy using some utility to compile the files on demand. That way the library does not necessary have to even reference the sass libraries, unless the runtime sass compilation is desired.
It would be great to have this feature as an option, thanks!

**Comment by w8tcha** [comment]
Sorry but I don't understand what you want. 

In order to convert the emails the sass need to be converted to css, and this happen on demand.

Or is your problem that the files are cached?

**Comment by Cherepoc** [comment]
What I want is to compile sass during development to produce css files, and then use the library to compile bootstrap templates using those css files in the runtime. So you weren't really incorrect when you said I want to use css instead of sass with your library, it's true. But during development I still want a tool to compile the sass files. As far as I understand the css files need specific structure/names. It would be great if there was a cli tool to just compile sass in a way the css will be ready for the project.

**Comment by w8tcha** [comment]
I released a new Version 1.5.2 you can now parse css files and skip the sass parsing

```c#
new BootstrapEmail(new ConfigStore { CssEmailPath = "emailtest.css", CssHeadPath = "Headtest.css" })
```

the head css is used to be placed inside the style tag inside the head tag. and the email css is used for in-lining. As sass compiler you can use any sass compiler you want. 

For example the `Citizen17.DartSass` library (nuget package) which is also used inside the BootstrapEmail library
